lib/Service/Rest.js

```diff
@@ -9,6 +9,12 @@ class ServiceRest extends CoreBase {
 		this.system.on('rest_get', (url, cb, extra_headers, extra_args) => {
 			this.debug('making request:', url)
 
+
+			if(!url.startsWith('http://') || !url.startsWith('https://')){
+				cb(true, { error: 'Url does not start with http:// or https://' })
```

Comment:
@Julusian should this technically be: `cb(true, { error: new Error('Url does not start with http:// or https://') })

Reply:
Yeah probably should be

Reply:
I'll update that then.

Reply:
@JTF4 are we still expecting this update?

Reply:
@krocheck yes. I got bogged down with some other projects. I'll probably update the commit tomorrow or the day after.

Reply:
@krocheck would it be worth promisifying the process? Or is that I thing for a future companion release?